### PR TITLE
Remove kwargs in elementwise layers and scale layer

### PR DESCRIPTION
**Pull request by sneaxiy**

### CODE REVIEW

**Review by reyoung — DISMISSED** [review]

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -6234,3 +6242,87 @@ def expand(x, expand_times, name=None):
         outputs={'Out': out},
         attrs={'expand_times': expand_times})
     return out
+
+
+def _elementwise_op(helper):
+    op_type = helper.layer_type
+    x = helper.kwargs.get('x', None)
+    y = helper.kwargs.get('y', None)
```
Comment (panyx0718) [inline python/paddle/fluid/layers/nn.py:6469]: seems simpler to just pass x and y into _elementwise_op

File: python/paddle/fluid/layers/nn.py [hunk 2]
```diff
@@ -6234,3 +6242,87 @@ def expand(x, expand_times, name=None):
...
+    assert x is not None, 'x cannot be None in {}'.format(op_type)
+    assert y is not None, 'y cannot be None in {}'.format(op_type)
+    axis = helper.kwargs.get('axis', -1)
+    use_mkldnn = helper.kwargs.get('use_mkldnn', False)
```
Comment (panyx0718) [inline python/paddle/fluid/layers/nn.py:6473]: same as above

**Review by panyx0718 — DISMISSED** [review]
Will be great to follow up with a test to make sure the APIs are added correctly

File: python/paddle/fluid/layers/nn.py [hunk 3]
```diff
@@ -6453,3 +6461,98 @@ def expand(x, expand_times, name=None):
...
+        outputs={'Out': out},
+        attrs={
+            'scale': float(scale),
+            'bias': float(bias),
+            'bias_after_scale': bias_after_scale
+        })
+    return out
+
+
+def elementwise_add(x, y, axis=-1, use_mkldnn=False, act=None):
```
Comment (qingqing01) [inline python/paddle/fluid/layers/nn.py]: Please add name for theses APIs.

**Review by qingqing01 — DISMISSED** [review]

File: python/paddle/fluid/layers/nn.py (hunk 3 above)
Comment (sneaxiy) [reply python/paddle/fluid/layers/nn.py]: Done

**Review by qingqing01 — DISMISSED** [review]

**Review by panyx0718 — APPROVED** [review]

**Review by qingqing01 — APPROVED** [review]

**Pull request merged**